.changeset/sweet-feet-study.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-linguist-backend': minor
+---
+
+**BREAKING**: Removing options from exported `linguistPlugin()` in favour of static config
```

Comment:
Should we mention this is only for the new backend system? I'm not sure if just mentioning `linguistPlugin()` makes that clear

Reply:
good point - yes!